src/backend/api/handlers/status.py

```diff
@@ -28,4 +29,10 @@ def status() -> Response:
     status["is_datafeed_down"] = ApiStatusFMSApiDown.get()
     status["down_events"] = down_events_list if down_events_list is not None else []
 
+    max_team_key = Team.query().order(-Team.team_number).fetch(1, keys_only=True)[0]
+    max_team_num = int(max_team_key.id()[3:])
+    max_team_page = int(max_team_num / 500)
```

Comment:
Might be worth making the page size a shared constant somewhere, and use it both here & in the API (so we can make sure they reference the same thing)